src/HOME4/home4_2.sql

```diff
@@ -0,0 +1,122 @@
+drop database if exists car;
+create database car;
+use car;
+
+
+CREATE TABLE owners(
+id int not null primary key auto_increment,
+name_of_owners varchar(30) not null,
+surname_of_owners varchar(30) not null,
+telephone integer not null,
+email varchar(30),
+town_id int not null,
+country_id int not null
```

Comment:
If we change `country_id ` only, we could get into state that there is no such `town_id`.

Owners should only be related to `towns` which related to `countries`.